referenceframe/frame.go

```diff
@@ -354,7 +354,7 @@ func (pf *translationalFrame) ProtobufFromInput(input []Input) *pb.JointPosition
 // Geometries returns an object representing the 3D space associeted with the translationalFrame.
 func (pf *translationalFrame) Geometries(input []Input) (*GeometriesInFrame, error) {
 	if pf.geometry == nil {
-		return nil, fmt.Errorf("frame of type %T has nil geometry", pf)
+		return NewGeometriesInFrame(pf.Name(), nil), nil
```

Comment:
curious why this is changing

Reply:
This cropped up when I was pulling geometries from the frame system to check names against; in our test with a gantry, it had no geometry, and thus errored.

We generally should not be forbidding the existence of frames that do not have geometries.